src/MembershipManager.php

```diff
@@ -214,6 +214,11 @@ public function getGroupIds(EntityInterface $entity, $group_type_id = NULL, $gro
       throw new \InvalidArgumentException('\Drupal\og\MembershipManager::getGroupIds() cannot be used for user entities. Use \Drupal\og\MembershipManager::getUserGroups() instead.');
     }
 
+    // This should only be called on group content types.
+    if (!$this->groupAudienceHelper->hasGroupAudienceField($entity->getEntityTypeId(), $entity->bundle())) {
+      throw new \InvalidArgumentException('Can only retrieve group IDs for group content entities.');
```

Comment:
Can config entities belong to a group? If so I'd go for:

```suggestion
      throw new \InvalidArgumentException('Can only retrieve group IDs for group entities.');
```

Reply:
Since the group is referenced on a entity reference field, all group content needs to be a content entity (since config entities are not fieldable). Also the name "group content" implies that it is a content entity :grin: 

Users are not considered to be group content, they are considered to be group members, and are managed not through a reference field, but through the `OgMembership` entity tables.

Reply:
I could be wrong here but isn't webform a bundlable (fieldable) config entity?
See https://www.drupal.org/project/webform/issues/3049023#comment-13073385